index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
do you that what this meta tags does ??

Reply:
great 👍 
 for more information you can check this  : https://www.google.com/amp/s/www.geeksforgeeks.org/explain-use-of-meta-tags-in-html/amp/